sdk/core/azure-core/src/main/java/com/azure/core/models/HttpResponseError.java

```diff
@@ -64,7 +64,7 @@ public String getMessage() {
      *
      * @return the target of this error.
      */
-    public String getTarget() {
+    String getTarget() {
```

Comment:
I was not able to attend the arch board review for this type. What's the reason for making this package private? If we do that then what's the purpose of having these fields? Is it so we can later expose them if required?

Reply:
Yes, the only two fields that are part of the API contract are message and code. So, starting with exposing only those two fields in the public API and we'll expose other fields if required later.